bin/flutterflow_cli.dart

```diff
@@ -130,6 +131,11 @@ ArgResults _parseArgs(List<String> args) {
       help: 'Generate the project with debug logging to be able to use '
           'FlutterFlow Debug Panel inside the DevTools.',
       defaultsTo: false,
+    )
+    ..addOption(
+      'environment-name',
+      help: '(Optional) Specify a environment name.',
+      defaultsTo: 'Production',
```

Comment:
I don't think it should have any default. If it's not set we should not send it to the server at all, right? Is so, also update README.

Reply:
The default name choice is to avoid breaking existing user's pipeline after some discussion in the dev environment channel. What do you think?